### ISSUE TITLE
Emotiv device to QTimeFreq viewer : incorect outputs stream dtype

### ISSUE THREAD

**Issue by Lx37** [pyacq/pyacq#67]
Trying to connect Emotiv signal output stream to QTimeFreq viewer gives an error on 
/stream/sharedmemstream.py", line 26 :
assert data.dtype == self.params['dtype']

signal type of Emotiv is 'int16' and self.params['dtype'] is 'float32',
even when specifying in 
emotivDevice.outputs['signals'].configure( ...,  dtype='int16',)
(exemple code here :  [emotiv_tfr.py.zip](https://github.com/pyacq/pyacq/files/1355612/emotiv_tfr.py.zip) )

is it an incorrect configuration of outputs streams in Emotiv device node ?

**Comment by samuelgarcia** [comment]
Ne fais pas ça:
```
emotivDevice.outputs['signals'].configure( ..., dtype='int16',)
```
car c'est déjà dans Emotiv._output_specs (en gros c'est en dur).

Dans ce cas, il va effectivement te falloir un pyacq.core.tools.StreamConverter.
C'est un node qui ne fais rien à part convertir les entrées/sorties.

**Comment by samuelgarcia** [comment]
closed by #68
